src/test_lib_json/main.cpp

```diff
@@ -2354,14 +2354,22 @@ JSONTEST_FIXTURE(CharReaderAllowSpecialFloatsTest, issue209) {
     JSONCPP_STRING in;
   };
   const TestData test_data[] = {
-    { __LINE__, true, "{\"a\":9}" },         { __LINE__, false, "{\"a\":0Infinity}" },
```

Comment:
These obviously need to all be on their own row. The EOL comments tell clang-format not to join them.

Reply:
+1